### ISSUE TITLE
The interactive polar plot animation's axis label won't scale. 

### ISSUE THREAD

**Issue by ewan1983** [matplotlib/matplotlib#16299]
### Bug report

**Bug summary**

In interactive polar plot animation, when I zoom in or zoom out, the plot itself does scale accordingly, but the r-axis label ticks stay the same, it won't scale accordingly.

**Code for reproduction**

```python
import numpy as np
from matplotlib import pyplot as plt
from matplotlib import animation

fig = plt.figure()
ax = fig.add_subplot(111,polar=True)
line, = ax.plot([], [], lw=2)

def animate(i):
    r = np.arange(0, 2, 0.001)
    theta = 2 * np.pi * r * i/77 + i/5
    line.set_data(theta, r)
    return line,

anim = animation.FuncAnimation(fig, animate, interval=20, blit=True)
```

**Matplotlib version**

  * Operating system: Win10
  * Matplotlib version: 3.1.0
  * Matplotlib backend (`print(matplotlib.get_backend())`): Qt5Agg
  * Python version:
  * Jupyter version (if applicable):
  * Other libraries:

**Comment by jklymak** [comment]
I think the blitting is supposed to work, that is the expected behaviour?  Did you try `blit=False`?

**Comment by ewan1983** [comment]
Yes, blitting is the culprit.
blit=False solves the problem.

**Comment by anntzer** [comment]
This is mostly fixed, even with blitting, as of master, by https://github.com/matplotlib/matplotlib/pull/16070; setting saner r limits with https://github.com/matplotlib/matplotlib/pull/16300 is even better.

**Comment by anntzer** [comment]
Fixed by #16070 and #16300.
